diag_manager/fms_diag_yaml.F90

```diff
@@ -269,39 +275,72 @@ subroutine diag_yaml_object_init
   integer              :: var_count        !< The current number of variables added to the diag_yaml obj
   integer              :: nvars            !< The number of variables in the current file
   integer, allocatable :: var_ids(:)       !< Ids of the variables in diag_table yaml
+  logical              :: is_ocean         !< Flag indicating if it is an ocean file
```

Comment:
I think that will be used in PP (@ceblanton). Not sure what the plan with it is.

Reply:
It might make sense to remove `realm` then. It was a half-idea that may not make any sense. Actually, I don't see it here to refresh my memory. Is it here?

Reply:
Currently, `realm` is just read in from the diag_table.yaml

https://github.com/NOAA-GFDL/FMS/blob/dmUpdate/diag_manager/fms_diag_yaml.F90#L77
https://github.com/NOAA-GFDL/FMS/blob/dmUpdate/diag_manager/fms_diag_yaml.F90#L359

We have no plans for it. We are probably just going to write it as a global_attribute in the file, which can then be used in PP?

Reply:
Thanks @uramirez8707 for the reminder. I still think the realm could be useful in PP, but wonder if it should be set in the diag_table. It is sort of more like the units or standard_name, unique for a given diagnostic, than an instruction for the diag manager (like the file units, frequency, regions). Still, having the optional ability to set this info in the diag_table and refer to it in the global attributes would be valuable.

Is there a default if not set, or just empty?

Reply:
the default is an empty string

Reply:
@ceblanton The units and standard_name are now also in the diag_table.yaml.